fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/system/ExportJobListener.java

```diff
@@ -0,0 +1,100 @@
+
+import com.ibm.fhir.bulkcommon.Constants;
+import com.ibm.fhir.bulkexport.common.CheckPointUserData;
+
+@Dependent
+public class ExportJobListener implements JobListener {
+    private static final Logger logger = Logger.getLogger(ExportJobListener.class.getName());
+
+    long currentExecutionStartTimeInMS;
+
+    @Inject
+    JobContext jobContext;
+
+    @Inject
+    @BatchProperty(name = Constants.IMPORT_FHIR_DATASOURCES)
+    String dataSourcesInfo;
+
+    public ExportJobListener() {
+        // do nothing
+    }
+
+
+    @Override
+    public void afterJob() {
+        long currentExecutionEndTimeInMS = TimeUnit.NANOSECONDS.toMillis(System.nanoTime());
+
+        // Used for generating response for all the import data resources.
+        @SuppressWarnings("unchecked")
+        List<CheckPointUserData> partitionSummaries = (List<CheckPointUserData>)jobContext.getTransientUserData();
+
+        JobOperator jobOperator = BatchRuntime.getJobOperator();
+        long totalJobExecutionMilliSeconds = 0;
+        // The job can be started, stopped and then started again, so we need to add them all to get the whole job execution duration.
+        for ( JobExecution jobExecution: jobOperator.getJobExecutions(jobOperator.getJobInstance(jobContext.getExecutionId()))) {
+            // For current execution, jobExecution.getEndTime() is either null or with wrong value because the current execution is not
+            // finished yet, so always use system time for both job execution start time and end time.
+            if (jobExecution.getExecutionId()  == jobContext.getExecutionId()) {
+                totalJobExecutionMilliSeconds += (currentExecutionEndTimeInMS - currentExecutionStartTimeInMS);
+            } else {
+                totalJobExecutionMilliSeconds += (jobExecution.getEndTime().getTime() - jobExecution.getStartTime().getTime());
+            }
+        }
+
+        // If the job is stopped before any partition is finished, then nothing to show.
+        if (partitionSummaries == null) {
+            return;
+        }
+
+        double jobProcessingSeconds = totalJobExecutionMilliSeconds / 1000.0;
+        jobProcessingSeconds = jobProcessingSeconds < 1 ? 1.0 : jobProcessingSeconds;
+
+        // log the simple metrics.
+        logger.info(" ---- Fhir resources exported in " + jobProcessingSeconds + "seconds ----");
+        logger.info("ResourceType \t| Exported");
+        int totalExportedFhirResources = 0;
+        List<String> resourceTypeSummaries = new ArrayList<>();
+        for (CheckPointUserData partitionSummary : partitionSummaries) {
+            logger.info(partitionSummary.getResourceTypeSummary() + "\t|"
+                  + partitionSummary.getTotalResourcesNum());
+            resourceTypeSummaries.add(partitionSummary.getResourceTypeSummary());
+            totalExportedFhirResources += partitionSummary.getTotalResourcesNum();
+        }
+
+        logger.info(" ---- Total: " + totalExportedFhirResources
```

Comment:
this seems like a good thing to have a log.fine

Reply:
this is the summary of the whole export job, so I used info instead of fine.